storage/badger/operation/common.go

```diff
@@ -133,6 +133,16 @@ func remove(key []byte) func(*badger.Txn) error {
 	}
 }
 
+func batchRemove(key []byte) func(writeBatch *badger.WriteBatch) error {
```

Comment:
```suggestion
// batchRemove removes the given key in a batch.
// It returns nil if the key is successfully removed
// It returns nil if the key does not exist
// It returns error if running into any exception
func batchRemove(key []byte) func(writeBatch *badger.WriteBatch) error {
```

Could you add a test case to verify the following case:
```
// It returns nil if the key does not exist
```